src/main/java/interfaces/IStorageService.java

```diff
@@ -0,0 +1,10 @@
+package interfaces;
+
+import java.util.Map;
+import model.Position;
+
+public interface IStorageService {
+    void put(Position position);
+
+    Map buy(String fruitName, int quantity);
```

Comment:
let's maybe make the return value `boolean` (if the operation was successful)

Reply:
done

Reply:
done